### ISSUE TITLE
Transitions - Custom easing is broken

### ISSUE THREAD

**Issue by taylornowotny** [webstudio-is/webstudio#2624]
- When you select Custom from the Easing menu, nothing happens. The select box should show "Custom".
 - I know we wanted the Easing select box to change to reflect the code value. But if a user deliberately selects Custom from the menu, the select box should show "Custom".
 8https://github.com/webstudio-is/webstudio/assets/86495896/30f93d43-e4e0-4f70-aefb-2274a66b7617
 
 - When you change the easing in code, the Easing value in the select box should reflect the change. It works for most values, but not any of the 3-word values like "Ease Out Sine" or "Ease In Quad" etc.
https://github.com/webstudio-is/webstudio/assets/86495896/12868ad6-cd11-4ca1-8636-3d9c2159bdf2

 - Canvas hotkeys are active while typing in the code input. I can't type the letter D becuase its duplicating items on the canvas instead.

**Comment by JayaKrishnaNamburu** [comment]
> When you change the easing in code, the Easing value in the select box should reflect the change - show the Easing value if it exists, show "Custom" if not. It works for most values, but not any of the 3-word values like "Ease Out Sine" or "Ease In Quad" etc. If you type one of these you get error state.

The value that users type in code box, is the same value that is added to the canvas or the projects. And that is why the value "Ease In Quad" actually doesn't show up in select when added in code. Because it's not a valid browser value. Unfortunately without hacks can't be achieved that. If it's really really needed though 😅  .

**Comment by taylornowotny** [comment]
@JayaKrishnaNamburu Ah understood, thanks for explaining.

Are you working on the other points?

**Comment by JayaKrishnaNamburu** [comment]
Yes, yes. I will go over the rest of them too. This one just initially got my eye 👍
